apps/trading/e2e/README.md

```diff
@@ -100,13 +100,13 @@ poetry run pytest
 Run a targeted test:
 
 ```bash
-poetry run pytest -k "test_name" -s
+poetry run pytest -k "test_name" -s --headed
 ```
 
 Run from anywhere:
 
 ```bash
-yarn trading:test -- "test_name" -s
+yarn trading:test -- "test_name" -s --headed
```

Comment:
do we want to encourage running in headed by default? Normally the opposite is true unless debugging. No strong opinion, just flagging

Reply:
Can revert, I just find when i'm running single tests i usually would want them headed.
can remove this and just make a note to show the command required to run in headed.

Reply:
up to you :)